doc/rst/build.rst

```diff
@@ -133,10 +133,25 @@ which can be done with the following commands:
 
    mkdir deps
    cd deps
-     wget https://github.com/hpc/libcircle/releases/download/v0.3/libcircle-0.3.0.tar.gz
-     wget https://github.com/llnl/lwgrp/releases/download/v1.0.4/lwgrp-1.0.4.tar.gz
-     wget https://github.com/llnl/dtcmp/releases/download/v1.1.4/dtcmp-1.1.4.tar.gz
-     wget https://github.com/libarchive/libarchive/releases/download/v3.5.1/libarchive-3.5.1.tar.gz
+
+     urls=(     https://github.com/hpc/libcircle/releases/download/v0.3/libcircle-0.3.0.tar.gz
+                https://github.com/llnl/lwgrp/releases/download/v1.0.4/lwgrp-1.0.4.tar.gz
+                https://github.com/llnl/dtcmp/releases/download/v1.1.4/dtcmp-1.1.4.tar.gz
+                https://github.com/libarchive/libarchive/releases/download/3.5.1/libarchive-3.5.1.tar.gz
+     )
+
+     for url in ${urls[*]}
+     do
+             wget $url
+             rc=$?
+             if [[ $rc -ne 0 ]];
+             then
+                     echo
+                     echo FAILED getting $url
+                     echo check for releases under $(echo $url | sed 's/releases.*/releases\//')
+                     exit 1
```

Comment:
Is it intended that the user put these commands in a script and then execute? I have often copy-pasted the commands directly from the site, so `exit 1` will be a logout in that case.

Reply:
That's a good point, I hadn't thought of the copy-paste.  I'll think of a different way to handle this.

Reply:
OK, I pushed an updated commit that uses if/else to stop executing once there is a failure to download.  It's now complex enough that I worry it obscures the basic process for installing the deps.
  @daltonbohning or @carbonneau1 do you have an opinion?

Reply:
In another channel, @carbonneau1 said "I don't think it's that bad. I think it is better organized."